dotnet-statsig/src/Statsig/Constants.cs

```diff
@@ -9,7 +9,7 @@ internal static class Constants
 
         public static int SERVER_MAX_LOGGER_QUEUE_LENGTH = 500;
         public static int SERVER_MAX_LOGGER_WAIT_TIME_IN_SEC = 60;
-        public static int SERVER_CONFIG_SPECS_SYNC_INTERVAL_IN_SEC = 10;
+        public static int SERVER_CONFIG_SPECS_SYNC_INTERVAL_IN_SEC = 60;
```

Comment:
Do we actually want to make this change? Right now max time you will need to wait for updated values to reflect is about 1 minute (70s), but with this change we will have to say 2 minutes